packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -281,7 +281,7 @@ function Cell(props) {
   const CellTag = isRowHeader ? 'th' : 'td';
 
   // eslint-disable-next-line react/forbid-dom-props
-  let cellContentComponent = <div className={cx('cell-content', theme.className)} style={{ height }}>{cellContent}</div>;
+  let cellContentComponent = <div className={cx('cell-content', theme.className)} style={{ 'min-height': height }}>{cellContent}</div>;
```

Comment:
we should also update the prop description of height : https://github.com/cerner/terra-framework/blob/1b2f1d728c349b3bda46f856981d4955174c134e/packages/terra-table/src/subcomponents/Cell.jsx#L106 

` String that specifies the min-height of the cell. Any valid CSS value is accepted.`